.github/workflows/ci.yml

```diff
@@ -109,14 +109,12 @@ jobs:
         #   In the other half we run the Opt profiles.
         #
         # We want the four parts to have similar runtimes.
-        BENCH_INCLUDE_EXCLUDE_OPTS: [
-          "--include cargo-0.60.0,stm32f4-0.14.0,webrender-2022",
-          "--exclude cargo-0.60.0,stm32f4-0.14.0,webrender-2022",
-        ]
-        PROFILES: [
-          "Check,Debug,Doc",
-          "Opt",
-        ]
+        BENCH_INCLUDE_EXCLUDE_OPTS:
+          [
+            "--include cargo-0.60.0,stm32f4-0.14.0,webrender-2022",
+            "--exclude cargo-0.60.0,stm32f4-0.14.0,webrender-2022",
+          ]
+        PROFILES: ["Check,Debug,Doc", "Opt"]
```

Comment:
`["Check,Debug,Doc","Opt"]` :)

Reply:
My editor automatically formats them, so I just remove the diff! Thanks.